skyportal/models/tag.py

```diff
@@ -9,25 +9,25 @@
 class ObjTagOption(Base):
     """A record of a list of tags that are available."""
 
-    tag_name = sa.Column(sa.String, nullable=False, doc="Tags list.", unique=True)
+    name = sa.Column(sa.String, nullable=False, doc="Tags list.", unique=True)
 
 
-ObjTags = join_model("obj_tags", ObjTagOption, Obj)
+ObjTags = join_model("obj_tag", ObjTagOption, Obj)
```

Comment:
Oh I meant the Python class mostly. It's possible that you need to write the name manually in that `join_model` method, so adding an `s` in there is fine in that case. I meant that the python class should not have an `s` as table names which are inferred from python classes use the class name + s. Not done anyway in your case since you specify the name explicitely in `join_model(...)` but even then let's not have the `s` in the python class name which would different from all other tables in the codebase = confusion.